index.js

```diff
@@ -183,6 +183,21 @@ WPCOM.prototype.sendRequest = function( params, query, body, fn ) {
 	return sendRequest.call( this, params, query, body, fn );
 };
 
+/**
+ * Re-export all the class types.
+ */
+
+WPCOM.Batch = Batch;
```

Comment:
It's because of the `babel-plugin-add-module-exports` plugin: you can't use `export default` _and_ other `export`s in the same file. A bit of a bummer but oh well.